### PR TITLE
Add support for vgpu-img development product

### PR DESCRIPTION
**Pull request by iartemenko**
Signed-off-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles.bbappend [hunk 1]
```diff
@@ -1,9 +1,10 @@
-PVRKM_URL = "git://github.com/xen-troops/pvr_km.git"
-BRANCH = "vgpu-dev"
+PVRKM_URL = "git@git.epam.com/epmd-aepr/pvr_km_vgpu_img.git"
+BRANCH = "master"
 SRCREV = "${AUTOREV}"
 
-SRC_URI_r8a7795 = "${PVRKM_URL};protocol=http;branch=${BRANCH}"
-SRC_URI_r8a7796 = "${PVRKM_URL};protocol=http;branch=${BRANCH}"
+
```
Comment (aanisov) [inline recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-kernel/kernel-module-gles/kernel-module-gles.bbappend]: empty line

File: recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module.bbappend [hunk 2]
```diff
@@ -0,0 +1,7 @@
+PVRUM_URL = "git@git.epam.com/epmd-aepr/pvr_um_vgpu_img.git"
+BRANCH = "master"
+SRCREV = "${AUTOREV}"
+
+
```
Comment (aanisov) [inline recipes-dom0/dom0-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module.bbappend]: an odd empty line

**Comment by iartemenko** [comment]
Updated, please review

File: recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/files/72-pvr-seat.rules.patch [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
- # License Strictly Confidential.
- #### ###########################################################################
- # Rules for PowerVR device nodes
--SUBSYSTEM=="drm", DEVPATH=="/devices/platform/soc/fd000000.gsx/drm/*", TAG+="seat", TAG+="master-of-seat", ENV{ID_AUTOSEAT}="1", ENV{ID_SEAT}="seat1"
-+SUBSYSTEM=="drm", DEVPATH=="/devices/platform/passthrough/fd000000.gsx/drm/*", TAG+="seat", TAG+="master-of-seat", ENV{ID_AUTOSEAT}="1", ENV{ID_SEAT}="seat1"
+ `# License Strictly Confidential.'
+ `#### ###########################################################################'
+ `# Rules for PowerVR device nodes'
+-`SUBSYSTEM=="drm", DEVPATH=="/devices/platform/soc/fd000000.gsx/drm/*", TAG+="seat", TAG+="master-of-seat", ENV{ID_AUTOSEAT}="1", ENV{ID_SEAT}="seat1"'
++`SUBSYSTEM=="drm", DEVPATH=="/devices/platform/passthrough/fd000000.gsx/drm/*", TAG+="seat", TAG+="master-of-seat", ENV{ID_AUTOSEAT}="1", ENV{ID_SEAT}="seat1"'
```
Comment (aanisov) [inline recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/files/72-pvr-seat.rules.patch]: Are you sure here will be /passthrough/?

Comment (andr2000) [reply recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/files/72-pvr-seat.rules.patch]: nobody sure at the moment as the feature is being developed now

Comment (aanisov) [reply recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/files/72-pvr-seat.rules.patch]: So what is the current criteria to state this stuff does what is required?

Comment (andr2000) [reply recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/files/72-pvr-seat.rules.patch]: no criteria. this is a boot strap code for now. will be updated once we have a vision on it

Comment (aanisov) [reply recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/files/72-pvr-seat.rules.patch]: > no criteria. this is a boot strap code for now. will be updated once we have a vision on it
keeping in mind this, lgtm, rb

**Pull request merged**